### PR TITLE
*: set default length for some types

### PR DESCRIPTION
**Pull request by XuHuaiyu**
PTAL
@zz-jason @hanfei1991

### CODE REVIEW

File: mysql/util.go [hunk 1]
```diff
@@ -30,11 +30,21 @@ func GetDefaultFieldLength(tp byte) int {
...
+		return 1
 	case TypeNewDecimal:
 		// See https://dev.mysql.com/doc/refman/5.7/en/fixed-point-types.html
 		return 10
-	case TypeBit, TypeBlob:
-		return -1
+	case TypeBlob:
+		return 65535
 	default:
 		//TODO: Add more types.
```
Comment (jackysp) [inline mysql/util.go]: Should we remove this comment?

Comment (jackysp) [inline mysql/util.go]: Should we remove this comment?

Comment (XuHuaiyu) [inline mysql/util.go]: Noop, there still remains some types not supported.

**Comment by XuHuaiyu** [comment]
PTAL @jackysp

**Pull request merged**